src/main/java/homework_2/bufferReaders/BufferReader.java

```diff
@@ -0,0 +1,45 @@
+package homework_2.bufferReaders;
+
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStreamReader;
+
+public class BufferReader {
```

Comment:
Did you name it bufferReader because BufferedReader already exists? Bad idea, confusing, I think. Better something like IOHelper